src/web/screens/createAccount.tsx

```diff
@@ -20,85 +13,38 @@ import { PortalRoute } from './routeUtils';
 export const AccountCreationRoutes: PortalRoute[] = [];
 
 function CreateAccount() {
-  const { participantTypes } = useLoaderData() as { participantTypes: ParticipantType[] };
   const { LoggedInUser, loadUser } = useContext(CurrentUserContext);
   const navigate = useNavigate();
-  const defaultFormData = {
-    canSign: true,
-  };
 
-  const onSubmit: SubmitHandler<CreateParticipantForm> = async (formData) => {
-    await CreateParticipant(formData, LoggedInUser!.profile);
+  const { participantTypes } = useLoaderData() as { participantTypes: ParticipantType[] };
+  const onSubmit = async (data: CreateParticipantForm) => {
+    const createResult = await CreateParticipant(data, LoggedInUser!.profile);
+    if ('errorStatus' in createResult) {
+      return createResult.messages;
+    }
     await loadUser();
-    navigate('/account/pending');
+    return navigate('/account/pending');
```

Comment:
what's the reason of return the navigate here?

Reply:
It's just returning void to satisfy the linter - I'll update the lint rule instead, it doesn't really make sense.